### DISCUSSION TITLE
logs of user modification

### DISCUSSION THREAD

**Discussion by quer0016** [JanssenProject/jans#8276] [Q&A]
Hello team Where can I see the data modification logs that are carried out on a user and its creation?

**Comment by ossdhaval** [comment]
Looping @jgomer2001 for comments.

**Comment by jgomer2001** [comment]
@quer0016 please describe the mechanism you are using to manage users in Jans

**Reply by quer0016** [reply]
adminui

**Comment by jgomer2001** [comment]
I see, in that case @syntrydy or @duttarnab may help you

**Reply by syntrydy** [reply]
@quer0016 all adminui logs are under **/var/log/adminui**, in there you have the base log file and the audit.
I think the one you are looking for is the **adminuiAudit.log**, to looks for audit relation to user modification, you can search in the file per resource: eg. **resource=person**.

**Comment by nynymike** [comment]
As @jgomer2001 says, you can add users in a few ways: 
* SCIM
* Config API
* via an Agama flow or interception script

For SCIM and Config API, there are the respective logs.

**Reply by quer0016** [reply]
In this case I am only interested in knowing where I can see the modifications and users created from the adminui

**Reply by syntrydy** [reply]
@quer0016 all adminui logs are under **/var/log/adminui**, in there you have the base log file and the audit.
I think the one you are looking for is the **adminuiAudit.log**, to looks for audit relation to user modification, you can search in the file per resource: eg. **resource=person**.

**Reply by quer0016** [reply]
I have this log:
 [qtp366004251-18] [ca.plugin.adminui.rest.logging.AuditLoggerResource] (AuditLoggerResource.java:35) - {headers={Content-Type=application/json}, userAction={client_id=2001.74718900-5632-48e4-9c33-0aeeef5eb4ef, author=User, status=succeed, action=FETCH, resource=person schema, message=, payload={}, date=2024-05-03T11:36:56.689Z}}

But I would like to know if it can be seen, which user made the modification, the user who modified it and what value was modified. Is that possible?

**Comment by ossdhaval** [comment]
Closing this as a stale discussion. Please comment if this is still a concern.
